src/RoadRegistry.BackOffice.Api/Extracts/DownloadExtractRequestBodyValidator.cs

```diff
@@ -0,0 +1,42 @@
+namespace RoadRegistry.BackOffice.Api.Extracts
+{
+    using System;
+    using FluentValidation;
+    using NetTopologySuite.Geometries;
+    using NetTopologySuite.IO;
+
+    public class DownloadExtractRequestBodyValidator : AbstractValidator<DownloadExtractRequestBody>
+    {
+        private readonly WKTReader _reader;
+
+        public DownloadExtractRequestBodyValidator(WKTReader reader) //TODO: Add logger to be able to log why parsing the WKT failed
+        {
+            _reader = reader ?? throw new ArgumentNullException(nameof(reader));
+
+            RuleFor(c => c.RequestId)
+                .NotEmpty().WithErrorCode("RequestIdMissing");
```

Comment:
Not sure we need to stick with these error codes since the currently implemented problem details does not use it.

Reply:
Removed